docs/tutorial/02-add-e2e-test.md

```diff
@@ -0,0 +1,41 @@
+# Step 2: Add E2E Test
```

Comment:
I think we should add here that, we're making an E2E test which will test the entire stack of the application. It fails for now, but we will learn how to do both the frontend and backend work all in 1 place to make it pass.

Reply:
Fixed